routers/init.go

```diff
@@ -107,7 +107,7 @@ func InitWebInstallPage(ctx context.Context) {
 }
 
 // InitWebInstalled is for global installed configuration.
-func InitWebInstalled(ctx context.Context) {
+func InitWebInstalled(ctx context.Context, extraDBPrepare func() error) {
```

Comment:
Because some Init service methods need to read databases data. This is different from production env and test env. In test env, all data are loaded from yml files. But in production env, all data in the databases before this method invoked.

Reply:
> Because some Init service methods need to read databases data.

Still can't understand. For example?